### PR TITLE
lib/model: Temp names from all platforms should be recognized as such

### PR DESCRIPTION
**Pull request by calmh**
### Purpose

Spring cleaning.

### CODE REVIEW

**Comment by AudriusButkevicius** [comment]
@st-review lgtm

**Comment by st-review** [comment]
@AudriusButkevicius: Noted! Need another LGTM or explicit merge command.

**Comment by calmh** [comment]
@st-review merge it

**Comment by st-review** [comment]
@calmh: Build status is `pending`. I'll wait until it goes green and then merge!

File: lib/model/tempname.go [hunk 1]
```diff
@@ -15,26 +15,41 @@ import (
...
 	}
 }
 
+// IsTemporary is true if the file name has the temporary prefix. Regardless
+// of the normally used prefix, the standard Windows and Unix temp prefixes
+// are always recognized as temp files.
 func (t tempNamer) IsTemporary(name string) bool {
-	return strings.HasPrefix(filepath.Base(name), t.prefix)
+	name = filepath.Base(name)
+	for _, prefix := range t.recognize {
```
Comment (plouj) [inline lib/model/tempname.go]: Is there any reason why this can't directly access the windowsTempPrefix and unixempPrefix constants defined at line 23?

**Comment by st-review** [comment]
:ok_hand: Merged as 48a229a0cdda6fa1fc9a4a482f591301e860183e. Thanks, @calmh!

**Pull request closed without merging**